cequel.gemspec

```diff
@@ -24,19 +24,20 @@ DESC
 
   s.files = Dir['lib/**/*.rb', 'templates/**/*', 'spec/**/*.rb', '[A-Z]*']
   s.test_files = Dir['spec/examples/**/*.rb']
-  s.required_ruby_version = '>= 2.0'
+  s.required_ruby_version = '>= 2.3'
 
   s.add_runtime_dependency 'activemodel', '>= 4.0'
+  s.add_runtime_dependency 'bigdecimal', '~> 3.0'
   s.add_runtime_dependency 'cassandra-driver', '~> 3.0'
```

Comment:
In a follow up MR, we'll switch this to use the fork that Jeremy is setting up

Reply:
Will do